client/app/components/packages/rwcds-form/with-action-no-action.hbs

```diff
@@ -0,0 +1,189 @@
+        @type="radio-group"
+        @attribute="dcpExistingconditions"
+        as |RadioGroup|
+      >
+        <RadioGroup
+          @options={{array
+            (hash label='Yes' code=true)
+            (hash label='No' code=false)
+          }}
+          @onChange={{fn (mut @form.saveableChanges.dcpDescribethenoactionscenario) ""}} as |value|
+        >
+          {{#if value}}
+            <Ui::Question as |Q|>
+              <Q.Prompt>
+                How did you determine this No-Action scenario?
+              </Q.Prompt>
+                <p>
+                  <span class="help-text">If Department of Building (DOB) approvals were granted that support the future No-Action Scenario assumed in the prior section, please indicate that in this summary, and if available, attach the DOB approved plans to this document.</span>
+                </p>
+              <Form.Field
+                @attribute="dcpHowdidyoudeterminethenoactionscenario"
+                @type="text-area"
+                @maxlength="1500"
+              />
+            </Ui::Question>
+          {{/if}}
+          {{#if (eq value false)}}
+            <Ui::Question as |Q|>
+              <Q.Prompt>
+                Description of No-Action scenario
+              </Q.Prompt>
+              <AdditionalText
+                @additionalTextTitle="Additional Guidance"
+              >
+                <p>
+                  Describe a future development scenario that is likely and would reasonably occur absent approval of the Proposed Action(s) (and other discretionary actions) by the Analysis Year, including:
+                  <p>
+                    <ul>
+                      <li>How all of the Development Sites(s) in question could reasonably be developed with regard to the underlying zoning district and any other applicable site controls</li>
+                      <li>The proposed Project’s/Development's overall gsf</li>
+                      <li>All proposed land uses (in gsf per land use)</li>
+                      <li>Overall height, number of stories</li>
+                      <li>Floor Area Ratio (F.A.R.) and lot coverage</li>
+                      <li>Number of dwelling units (and number of affordable dwelling units, as well as the mechanism for providing affordable units, e.g., MIH)</li>
+                      <li>Number of parking spaces</li>
+                      <li>New curb cuts (where applicable)</li>
+                      <li>Development trends in the neighborhood</li>
+                      <li>Site ownership, etc</li>
+                    </ul>
+                  </p>
+                  If a prior environmental review was conducted on the Project/Development Site, and the Projected or Potential Development Sites were assumed as soft sites in the prior environmental review materials, it may be appropriate to assume that the prior project’s With-Action Scenario is this Project’s/Development’s future No-Action Scenario. 
+                </p>
```

Comment:
Here there's a list inside a paragraph inside a paragraph. This is invalid HTML.